substrate/client/transaction-pool/src/graph/pool.rs

```diff
@@ -60,27 +61,36 @@ pub type TransactionFor<A> = Arc<base::Transaction<ExtrinsicHash<A>, ExtrinsicFo
 pub type ValidatedTransactionFor<A> =
 	ValidatedTransaction<ExtrinsicHash<A>, ExtrinsicFor<A>, <A as ChainApi>::Error>;
 
+/// The priority of request to validate the transaction.
+#[derive(PartialEq, Copy, Clone)]
+pub enum ValidateTransactionPriority {
```

Comment:
nit: When looking at this, its not clear to me which of the variants now has higher priority. At least the enum name implies that one has higher priority over the other. Should go into a comment here.